### PR TITLE
Add map to show details of fellows

### PR DESCRIPTION
**Pull request by NBNARADHYA**
- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

Implements #12

### CODE REVIEW

**Comment by riteshsp2000** [comment]
This is a WIP PR... 

Work to be done:
- Configure markers on the map as per user location
- Create Card UI for the user (to be shown when clicked on the marker)

**Comment by riteshsp2000** [comment]
@wrussell1999 @yashika51 

This fixes issue #13 as well. The card that I created pops up after clicking on the marker. It shows the fellow name, description, technologies and socials.

File: _config.yml [hunk 1]
```diff
@@ -10,4 +10,6 @@ github_username:  mlh-fellowship
 sass:
   sass_dir: _sass
 
-icon: /assets/img/icon.jpg
\ No newline at end of file
+icon: /assets/img/icon.jpg
+
+accessToken: pk.eyJ1IjoibmJuYXJhZGh5YSIsImEiOiJja2xpdjc0eXQwMHY5MnhwNTVmbjlwY2phIn0.pS57OxYhcCZegQr4psnEdg
```
Comment (yashika51) [inline _config.yml:15]: No secret tokens in main please. Use .env file.

File: _sass/map.scss [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 18px;
+  font-family: "Poppins", sans-serif;
+
+  margin: 0px;
+  margin-top: 5px;
+  padding: 0px;
+  text-align: left;
+}
```
Comment (yashika51) [inline _sass/map.scss:104]: I feel the description text looks very light, maybe experiment a bit to make it more clearly visible.

File: _data/fellows.yml [hunk 3]
```diff
@@ -1,15 +1,62 @@
...
+    y: 118
+  technologies:
+    - node
+    - typescript
+    - graphql
+    - express
+  socials:
+    twitter: https://twitter.com/aaronosher
+    linkedin: https://www.linkedin.com/in/aaronosher/
+    github: https://github.com/aaronosher
```
Comment (yashika51) [inline _data/fellows.yml:46]: I can only see one pin in the UK when we have two people from same region, why is it not appearing in the map? Is it just for me?

**Review by yashika51 — CHANGES_REQUESTED** [review]

File: _data/fellows.yml [hunk 4]
```diff
@@ -1,15 +1,62 @@
...
   location: New York City, USA
+  coordinates:
+    x: 28
+    y: 77
+  technologies:
+    - java
+    - spring-boot
+    - postgre
+  socials:
+    linkedin: https://www.linkedin.com/in/amanda-d-avria-287a017/
```
Comment (yashika51) [inline _data/fellows.yml:29]: Clicking on any pin other than Will's pin doesn't display any information. This is strange....

**Review by yashika51 — CHANGES_REQUESTED** [review]

File: _data/fellows.yml (hunk 3 above)
Comment (NBNARADHYA) [reply _data/fellows.yml:46]: The fellows data has been modified for now, and their location coordinates don't exactly map correctly to their actual places. So pins are not accurate for now.

File: _data/fellows.yml (hunk 4 above)
Comment (NBNARADHYA) [reply _data/fellows.yml:29]: It's working for me. I am not really sure why it doesn't for you.

File: _sass/map.scss (hunk 2 above)
Comment (riteshsp2000) [reply _sass/map.scss:104]: Okay sure... I'll try and experiment with few colors

File: _data/fellows.yml (hunk 3 above)
Comment (riteshsp2000) [reply _data/fellows.yml]: This looks nice!

Comment (yashika51) [reply _data/fellows.yml]: Great job @NBNARADHYA!

**Review by yashika51 — APPROVED** [review]

**Pull request merged**